src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentsService.java

```diff
@@ -5,6 +5,8 @@
 
     byte[] getLegalSealedClaim(String externalId, String authorisation);
 
+    byte[] getSealedClaim(String externalId, String authorisation);
```

Comment:
Do we need this new method? I think rather than having two different methods(`getLegalSealedClaim ` & `getSealedClaim `) here, we can have just one method probably called `getSealedClaim ` and inside it's implementation we can distinguish based on `claim.getClaimData().isClaimantRepresented()`.

We can rename `LegalSealedClaimPdfService` to `SealedClaimPdfService` which creates both PDFs. 
```
public byte[] createPdf(Claim claim) {
 if (claim.getClaimData().isClaimantRepresented()) {
 return pdfServiceClient.generateFromHtml(
            documentTemplates.getLegalSealedClaim(),
            legalSealedClaimContentProvider.createContent(claim)
        );
} else {
 return <CITIZEN_PDF_GENERATED>;
}
```